scripts/xml_to_csv_table_list.py

```diff
@@ -0,0 +1,29 @@
+import xml.etree.ElementTree as ET
```

Comment:
Why do we need this script? Google sheet is the source of truth right?

Reply:
To generate that google sheet that will be the Source of truth.